### PR TITLE
Gender neutral language

### PR DESCRIPTION
**Pull request by brianteeman**
A few quick changes to some strings and comment blocks to ensure gender neutral language

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_system_logout.sys.ini [hunk 1]
```diff
@@ -3,5 +3,5 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_SYSTEM_LOGOUT_XML_DESCRIPTION="The system logout plugin enables Joomla to redirect the user to the home page if he chooses to logout while he is on a protected access page."
+PLG_SYSTEM_LOGOUT_XML_DESCRIPTION="The system logout plugin enables Joomla to redirect the user to the home page if they choose to logout while they are on a protected access page."
```
Comment (piotr-cz) [inline administrator/language/en-GB/en-GB.plg_system_logout.sys.ini]: Should it not be _..users to the home page if they..._ ?

**Comment by piotr-cz** [comment]
I have tested this item :white_check_mark: successfully on 3688d4878fdcb7b2069564d885855d62706b4bb0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

File: administrator/language/en-GB/en-GB.plg_system_logout.sys.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_system_logout.sys.ini]: No

**Comment by grhcj** [comment]
I have tested this item :white_check_mark: successfully on 3688d4878fdcb7b2069564d885855d62706b4bb0<br /><br />On review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

**Comment by brianteeman** [comment]
RTC thanks <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

**Comment by franzpeter** [comment]
Administrator should be changed too. It is not gender neutral.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

**Comment by brianteeman** [comment]
It is in english<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

**Comment by franzpeter** [comment]
but not in other languages<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

**Comment by franzpeter** [comment]
I think exchanging administrator with administration would work for all languages.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10854">issues.joomla.org/joomla-cms/10854</a>.</sub>

**Comment by ggppdk** [comment]
@brianteeman  , some examples

----

> If options['autoregister'] is true, if the user doesn't exist yet **they will** be created

**Could be ?**:
If options['autoregister'] is true, if the user doesn't exist yet, **then user will** be created

----

> The system logout plugin enables Joomla to redirect the user to the home page **if they choose** to logout while they are on a protected access page.

**Could be ?**:

The system logout plugin enables Joomla to redirect the user to the home page **when user chooses** to logout **while being** on a protected access page.

----

English is not my native language, but using "user" instead of "they" is more proper / natural ?

i am just asking you and anyone else reading this thread

**Comment by brianteeman** [comment]
> English is not my native language, but using "user" instead of "they" is
more proper / natural ?

Not in english

> I think exchanging administrator with administration would work for all
languages.

Not in english - one is a person the other is a task



On 14 July 2016 at 12:11, Georgios Papadakis <notifications@github.com>
wrote:

> @brianteeman <https://github.com/brianteeman> , some examples
> ------------------------------
>
> If options['autoregister'] is true, if the user doesn't exist yet *they
> will* be created
>
> *Could be ?*:
> If options['autoregister'] is true, if the user doesn't exist yet, *then
> user will* be created
> ------------------------------
>
> The system logout plugin enables Joomla to redirect the user to the home
> page *if they choose* to logout while they are on a protected access page.
>
> *Could be ?*:
>
> The system logout plugin enables Joomla to redirect the user to the home
> page *when user chooses* to logout *while being* on a protected access
> page.
> ------------------------------
>
> English is not my native language, but using "user" instead of "they" is
> more proper / natural ?
>
> i am just asking you and anyone else reading this thread
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/10854#issuecomment-232635808>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABPH8WYQra0Uux0Ng0xBYkgpFQffU7-Hks5qVhlygaJpZM4I4O1L>
> .
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Pull request merged**